packages/react-css-utilities/README.md

```diff
@@ -0,0 +1,72 @@
+# `@shopify/react-css-utilities`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-css-utilities.svg)](https://badge.fury.io/js/%40shopify%2Freact-css-utilities.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-css-utilities.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/react-css-utilities.svg)
+
+A set of react styling related utilities.
+
+## Installation
+
+```bash
+$ yarn add @shopify/react-css-utilities
+```
+
+## Usage
+
+Both `classnames` and `variationName` utilities should ideally be use in combination with a `scss` styles sheet and [`css module`](https://github.com/css-modules/css-modules) setup to generate class name for react component.
```

Comment:
A few comments:

- You jump right into this without even saying that these are two functions exported by the package
- `scss` here doesn't really matter
- `CSS` (uppercase)
- Again, I don't think React is relevant here.

Reply:
Also it's `classNames` from the library

Reply:
I think you should also link to the `classname` library, you don't really say what that function even does (removes falsy values, etc)